examples/vue-example-app/src/components/utils/getData.js

```diff
@@ -1,7 +1,7 @@
 import axios from 'axios'
 import sensors from './sensors'
 
-const ABS_DATA_BASE_URL = 'https://proxy-myvic-components-production.lagoon.vicsdp.amazee.io/abs/sdmx-json/data/'
+const ABS_DATA_BASE_URL = 'https://proxy.production.myvic-components.sdp2.sdp.vic.gov.au/abs/sdmx-json/data/'
```

Comment:
This should work in production but you'll need to disable CORS for it to work properly